Update README.md (#16)
omasabdullah/omasabdullah: README.md

### Hi there 👋
who goes there

> * Great leaders inspire greatness in others.

Author: ```suggestion
* niko jr.
```

> who goes there
> * Great leaders inspire greatness in others.
> * Belief is not a matter of choice, but of conviction.

Author: ```suggestion
who goes there
* Great leaders inspire greatness in others.
This is weird
```
